tests/test_jobs_and_serve.py

```diff
@@ -254,7 +254,7 @@ def test_cancel_on_jobs_controller(self, _mock_cluster_state,
                                        _mock_jobs_controller):
         cli_runner = cli_testing.CliRunner()
         result = cli_runner.invoke(cli.cancel, [jobs.JOB_CONTROLLER_NAME, '-a'])
-        assert result.exit_code == 1
+        assert result.exit_code == 2
```

Comment:
Is this change intended?

Reply:
Good catch! Changed it to `click.UsageError.exit_code` for better readability